src/libpmem2/pmem2_utils_linux.c

```diff
@@ -172,3 +172,108 @@ pmem2_device_dax_size_from_stat(const os_stat_t *st, size_t *size)
 	LOG(4, "device size %zu", *size);
 	return 0;
 }
+
+/*
+ * pmem2_device_dax_alignment_from_stat -- checks the alignment of a given
+ * dax device from given stat structure
+ */
+int
+pmem2_device_dax_alignment_from_stat(const os_stat_t *st, size_t *alignment)
```

Comment:
we can't just copy code from one place to another - old instance should call the new one